packages/client-app/index.html

```diff
@@ -26,7 +26,7 @@
   <body>
     <div
       id="app"
-      class="app"></div>
+      class="app scalar-app scalar-client"></div>
```

Comment:
Do we want a class of `.app` at all?

Reply:
not really, don't think it does anything